sandpack-react/package.json

```diff
@@ -54,17 +54,17 @@
     "codesandbox-import-util-types": "^2.2.3",
     "lodash.isequal": "^4.5.0",
     "lz-string": "^1.4.4",
-    "react-devtools-inline": "4.4.0",
+    "react-devtools-inline": "4.25.0",
```

Comment:
I'm guessing the version bump added a dependency on an unreleased/unstable React API.